### PR TITLE
Add local clocks to clocks table and definitions

### PR DESCRIPTION
**Pull request by andreas-junghanns**
As agreed during today´s FMI design meeting, I created this PR to explicitly define local clocks and their restrictions.

### CODE REVIEW

File: docs/2_2_common_mechanisms.adoc [hunk 1]
```diff
@@ -759,6 +764,12 @@ is activated with <<fmi3SetClock>> by the importer in <<EventMode>>, or triggers
 Triggered output Clock::
 is activated within the FMU and the importer must call <<fmi3GetClock>> in <<EventMode>> or in <<ClockUpdateMode>>.
 
+Triggered local Clock::
+is activated within the FMU and the importer must call <<fmi3GetClock>> in <<EventMode>> or in <<ClockUpdateMode>>.
```
[the remark this answers is not in the document]
Comment (andreas-junghanns) [reply docs/2_2_common_mechanisms.adoc:768]: thanks! Done.

[the remark this answers is not in the document]
Comment (andreas-junghanns) [reply docs/2_2_common_mechanisms.adoc]: Darn, missed that, sorry. Done.

[the remark this answers is not in the document]
Comment (andreas-junghanns) [reply docs/2_2_common_mechanisms.adoc]: @PTaeuberDS : ok now? Please approve!

**Review by PTaeuberDS — APPROVED** [review]

**Comment by andreas-junghanns** [comment]
Anyone else willing to approve or critique this PR? Please do so until the end of business today.

**Pull request merged**